lib/re_glob.ml

```diff
@@ -131,20 +117,150 @@ let explode str =
   in
   List.rev (fst (expl false 0 0 [] [""]))
 
-let glob ?anchored ?(period = true) ?(expand_braces = false) s =
-  let init_state =
-    if period
-    then Beg
-    else Mid
+module To_re = struct
```

Comment:
Why do you need this `To_re` module? Seems like it's just grouping stuff but it's not obvious to me why that's necessary.